content/hi/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -21,66 +21,66 @@ sitemap:
 कुबेरनेट्स नाम ग्रीक से उत्पन्न हुआ है, जिसका अर्थ है हेल्समैन या पायलट।
 K8s एक संक्षिप्त नाम के रूप में "K" और "s" के बीच आठ अक्षरों को गिनने का परिणाम है। Google ने 2014 में कुबेरनेट्स प्रोजेक्ट को ओपन-सोर्स किया। कुबेरनेट्स [गूगल के 15 से अधिक वर्षों के अनुभव](/blog/2015/04/borg-predecessor-to-kubernetes/) को समुदाय से सर्वोत्तम नस्ल के विचारों और प्रथाओं के साथ बड़े पैमाने पर उत्पादन कार्यभार को जोड़ती है।
 
-## काल में वापस जाना
+## समय में वापस जाना
 
-आइए काल में वापस जाकर एक नज़र डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है।
+आइए वक्त में पीछे जाकर एक नज़र डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है।
 
 ![डिप्लॉयमेंट का विकास](/images/docs/Container_Evolution.svg)
 
 **पारंपरिक डिप्लॉयमेंट युग:**
-प्रारंभ में, संगठनों ने भौतिक (physical) सर्वरों पर एप्लिकेशन चलाए। भौतिक (physical) सर्वर में एप्लिकेशनो के लिए संसाधन सीमाओं को परिभाषित करने का कोई तरीका नहीं था, और इससे संसाधन आवंटन समस्याएं उत्पन्न हुईं। उदाहरण के लिए, यदि एक से अधिक एप्लिकेशने एक भौतिक सर्वर पर चलते हैं, तो ऐसे उदाहरण हो सकते हैं जहां एक एप्लिकेशन अधिकांश संसाधनों को ले लेगा, और इसके परिणामस्वरूप, अन्य एप्लिकेशने खराब प्रदर्शन करेंगे। इसका एक समाधान यह होगा कि प्रत्येक एप्लिकेशन को एक अलग भौतिक सर्वर पर चलाया जाए। लेकिन यह पैमाना नहीं था क्योंकि संसाधनों का कम उपयोग किया गया था, और संगठनों के लिए कई भौतिक सर्वरों को बनाए रखना महंगा था।
+प्रारंभ में, संगठनों ने भौतिक (physical) सर्वरों पर एप्लिकेशन चलाए। भौतिक सर्वर में एप्लिकेशनो के लिए संसाधन सीमाओं को परिभाषित करने का कोई तरीका नहीं था, और इससे संसाधन आवंटन समस्याएं उत्पन्न हुईं। उदाहरण के लिए, यदि एक से अधिक एप्लिकेशने एक भौतिक सर्वर पर चलते हैं, तो ऐसे परिस्थिति हो  सकती हैं जहाँ एक एप्लिकेशन अधिकांश संसाधनों को ले लेगा, और इसके परिणामस्वरूप, अन्य एप्लिकेशने का प्रदर्शन खराब हो सकता है। इसका एक समाधान यह होगा कि प्रत्येक एप्लिकेशन को एक अलग भौतिक सर्वर पर चलाया जाए। लेकिन यह पैमाना नहीं था क्योंकि संसाधनों का कम उपयोग किया गया था, और संगठनों के लिए कई भौतिक सर्वरों को बनाए रखना महंगा था।
 
-**वर्चुअलाइज्ड डिप्लॉयमेंट युग:**  एक समाधान के रूप में, वर्चुअलाइजेशन पेश किया गया था। यह आपको एक भौतिक सर्वर के सीपीयू (CPU) पर कई वर्चुअल मशीन (वीएम) चलाने की अनुमति देता है। वर्चुअलाइजेशन एप्लिकेशनो को वीएम (VM) के बीच अलग-थलग करने की अनुमति देता है और सुरक्षा का एक स्तर प्रदान करता है क्योंकि एक एप्लिकेशन की जानकारी को दूसरे एप्लिकेशन द्वारा स्वतंत्र रूप से एक्सेस नहीं किया जा सकता है।
+**वर्चुअलाइज्ड डिप्लॉयमेंट युग:**  एक समाधान के रूप में, वर्चुअलाइजेशन पेश किया गया था। यह आपको एक भौतिक सर्वर के सीपीयू (CPU) पर कई वर्चुअल मशीन (वीएम) चलाने की अनुमति देता है। वर्चुअलाइजेशन एप्लिकेशनो को वीएम (VM) के बीच  अलगाव उपलब्ध करता है और सुरक्षा का एक स्तर प्राप्त करता है क्योंकि एक एप्लिकेशन की जानकारी को दूसरे एप्लिकेशन द्वारा स्वतंत्र रूप से प्राप्त नहीं किया जा सकता है।
```

Comment:
```suggestion
**वर्चुअलाइज्ड डिप्लॉयमेंट दौर:**  एक समाधान के रूप में, वर्चुअलाइजेशन पेश किया गया था। यह आपको एक भौतिक सर्वर के सीपीयू (CPU) पर कई वर्चुअल मशीन (वीएम) चलाने की अनुमति देता है। वर्चुअलाइजेशन एप्लिकेशनो को वीएम (VM) के बीच अलगाव उपलब्ध करता है और सुरक्षा का एक स्तर प्राप्त करता है, क्योंकि एक एप्लिकेशन की जानकारी को दूसरे एप्लिकेशन द्वारा स्वतंत्र रूप से प्राप्त नहीं किया जा सकता है।
```

Reply:
Virtualized deployment era vs Virtualized deployment period. what is better

Reply:
"Era" can also mean 'period' and "Era" can also mean दौर, युग 
![Screenshot (727)](https://github.com/kubernetes/website/assets/116108975/960d7732-9a7f-4b34-a045-1ce851a0bdcc)

 "युग"seems odd to me here so, i suggested "दौर" as alternative

Reply:
that is what I am asking  ,you want me to change era into period. 
![image](https://github.com/kubernetes/website/assets/71877143/94846881-051e-4c93-8c46-b0c3b6d8f766)

Reply:
I mean in english docs "era" word is fine, Now don't say like 'era' means युग and 'period' means दौर, 'दौर' word can also be used in context of 'era'.

In hindi, both of these words can be used in replacement of "era". But, i suggest using "दौर"